app/cdap/components/Home/HomeActions/ActionConfig.ts

```diff
@@ -90,7 +90,7 @@ export const ActionConfig: IAction[] = [
     links: [
       {
         label: T.translate(`${PREFIX}.Metadata.linkLabel`).toString(),
-        url: '/cdap/ns/:namespace/metadata',
+        url: '/metadata/ns/:namespace',
```

Comment:
Default metadata urls should point to angular metadata page since the react work is not completed yet.

Reply:
Oh I remember this one... The metadata api was changed quite some time ago